src/modules/create_append.test.js

```diff
@@ -0,0 +1,13 @@
+import createAppendTask from './create_append.js';
+
+test('Add new item to the list', () => {
+  document.body.innerHTML = `
+    <div>
+      <ul id="list-tasks">          
+      </ul>
+    </div>`;
+  const element = document.getElementById('list-tasks');
+  createAppendTask('task1', false, element);
+  const list = document.querySelectorAll('.task-item');
+  expect(list).toHaveLength(1);
+});
```

Comment:
- [ ] [REQUIRED] Great job team, however, we need to group our tests using the the `describe()` method, this breaks your test suite into components. Depending on your test strategy, you might have a describe for each function. Take a look at this example 
```
describe('my beverage', () => {
  test('is delicious', () => {
    expect(myBeverage.delicious).toBeTruthy();
  });

  test('is not sour', () => {
    expect(myBeverage.sour).toBeFalsy();
  });
});
```

Kindly, do this for other tests as well. Happy coding 👍🏽

Reply:
Thank you for your feedback, we didn't see this requirement 👍